runtimes/protocol/getConfigurationFromServer.ts

```diff
@@ -0,0 +1,13 @@
+import { AutoParameterStructuresProtocolRequestType, LSPAny } from './lsp'
+
+export const getConfigurationFromServerRequestType = new AutoParameterStructuresProtocolRequestType<
+    GetConfigurationFromServerParams,
+    LSPAny,
+    never,
+    void,
+    void
+>('aws/getConfigurationFromServer')
+
+export interface GetConfigurationFromServerParams {
```

Comment:
nit, can be done in follow-up: let's start adding documentation to protocol changes to explain purpose of params like this for servers and clients developers.